packages/dx-react-grid/docs/reference/table-column-resizing.md

```diff
@@ -12,10 +12,23 @@ A plugin that manages table column widths.
 
 Name | Type | Default | Description
 -----|------|---------|------------
-columnWidths | { [columnName: string]: number } | | Specifies column widths.
-defaultColumnWidths | { [columnName: string]: number } | {} | Specifies initial column widths in the uncontrolled mode.
+columnWidths | Array&lt;[ColumnWidths](#ColumnWidths)&gt; | | Specifies column widths.
+defaultColumnWidths | Array&lt;[ColumnWidths](#ColumnWidths)&gt; | [] | Specifies initial column widths in the uncontrolled mode.
 onColumnWidthsChange | (nextColumnWidths: { [columnName: string]: number }) => void | | Handles column width changes.
 
+## Interfaces
+
+### ColumnWidths
```

Comment:
We should find a good name for this.

Reply:
ColumnWidth or ColumnWidthsItem ?

Reply:
Maybe ColumnWidth. @viterobk, can you help?

Reply:
May be something like ColumnWidthInfo?

Reply:
We don't have any names with *Info. But *Info seems good :)

Reply:
Rename ColumnWidths -> ColumnWidthInfo